src/opera_rtc_s1_browse/create_browse.py

```diff
@@ -17,6 +18,28 @@
 s3 = boto3.client('s3')
 
 
+def now() -> datetime.datetime:
+    return datetime.datetime.utcnow()
```

Comment:
I recall `utcnow` being deprecated in favor of `datetime.now(tz=timezone.utc)`, but this might be our best option for getting UTC time without a timezone, so that `now().isoformat(timespec="seconds")}Z` works as intended?

Reply:
We could also use `replace('+00:00', 'Z')` as suggested by https://stackoverflow.com/a/42777551/8763290

Reply:
Yes, [`utcnow` is deprecated in `3.12`](https://docs.python.org/3/library/datetime.html#datetime.datetime.utcnow) so we may want to try a different approach to future-proof ourselves.